### PR TITLE
Update integration sample - Shopify

### LINKED ISSUE #144: Update integration sample - Shopify
### Brief bug description

After the [rebranding from Kentico Cloud to Kentico Kontent](https://kontent.ai/blog/moving-to-caas-with-kentico-kontent) there are still some samples that use old naming in branches named `samples/*`.

Until the end of October endpoint on domain `kenticocloud.com` stop working and there will be redirect performed:
* https://docs.kontent.ai/reference/api-changelog#a-api-urls-changed

#### Transfer changes made in https://github.com/Kentico/kontent-sample-app-react/pull/140 to the [**`samples/shopify`**](https://github.com/Kentico/kontent-sample-app-react/tree/samples/shopify) branch

* Update dependency from `kentico-cloud-delivery` to `@kentico/kontent-delivery`
    * adjust the breaking changes
* Update dependency `@chevtek/react-spinners` to `@simply007org/react-spinners`
   * no breaking changes
* Rename Kentico Cloud to Kentico Kontent
* adjust `travis.yml` to use the correct domain for deployment

### PR DESCRIPTION
**Pull request by obulaworld**
### Motivation

Fixes #144

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ script:
 deploy:
   provider: surge
   project: ./build/
-  domain: https://kentico-cloud-sample-app-react-shopify.surge.sh
+  domain: https://kontent-sample-app-react.surge.sh
```
Comment (Simply007) [inline .travis.yml]: Please keep the `-shopify` suffix

```suggestion
  domain: https://kontent-sample-app-react.surge-shopify.sh
```

**Comment by obulaworld** [comment]
> Could you also include these changes from #140
> 
> * Removal of `/examples/*` and `/docs/*`
> * Adjust the `value` property (i.e. like in https://github.com/Kentico/kontent-sample-app-react/pull/140/files#diff-32f4f85fc84ccaf9e994c43c71ec13bcR76)

Hi @Simply007 , The `src/Pages/Home.js` for the `samples/shopify` branch is different from the master branch and does not have a `home`variable defined. Should this be changed to match the `src/Pages/Home.js` in the master?

**Comment by Simply007** [comment]
May I ask you, how did you find this issue? Repo? Is that part of Hacktoberfest challenge? 😄

**Comment by obulaworld** [comment]
> May I ask you, how did you find this issue? Repo? Is that part of Hacktoberfest challenge? 😄

Yes it is

**Pull request merged**